Python/Airflow/InitializeAirflow.sh

```diff
@@ -0,0 +1,48 @@
+declare P_ENVIRONMENT=$1
```

Comment:
>P [](start = 8, length = 1)

What's the P signify?

Reply:
I'd been using it to indicate parameters passed into the script. Not super necessary in this script because of it's length, though.